flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/runtime/batch/sql/join/LookupJoinITCase.scala

```diff
@@ -30,19 +30,15 @@ import org.apache.flink.types.Row
 
 import org.assertj.core.api.Assertions.assertThat
 import org.assertj.core.api.IterableAssert.assertThatIterable
-import org.junit.{After, Assume, Before, Test}
 import org.junit.Assert.assertEquals
```

Comment:
We should remove the dependency of Junit 4, for example the `assertEquals`

Reply:
Done.